src/devices/Bmx280/BmxBase.cs

```diff
@@ -6,44 +6,37 @@
 //Formulas and code examples can also be found in the datasheet http://www.adafruit.com/datasheets/BST-BMP280-DS001-11.pdf
 
 using System;
+using System.Buffers.Binary;
 using System.Device.I2c;
 using System.Threading.Tasks;
-using System.Buffers.Binary;
 
-namespace Iot.Device.Bmp280
+namespace Iot.Device.Bmx280
 {
-    public class Bmp280 : IDisposable
+    public class BmxBase : IDisposable
     {
-        private const byte Signature = 0x58;
+        internal I2cDevice _i2cDevice;
+        internal byte _signature;
+        internal bool _initialized = false;
+        internal CommunicationProtocol _communicationProtocol;
+        internal CalibrationData _calibrationData;
 
-        private I2cDevice _i2cDevice;
-        private readonly CommunicationProtocol _communicationProtocol;
-        private bool _initialized = false;
-        private readonly CalibrationData _calibrationData;
         /// <summary>
         /// The variable _temperatureFine carries a fine resolution temperature value over to the
         /// pressure compensation formula and could be implemented as a global variable.
         /// </summary>
-        private int _temperatureFine;
+        internal int _temperatureFine;
```

Comment:
Why internal? I would have them be protected if needed by the subclasses, but if not I would keep as private.

Reply:
if going with protected then use naming convention as with public (`TemperatureFine`)

Reply:
I'll go for protected then. Can be useful for other sensors from the same familly. Won't change much for this PR.